src/test/java/core/basesyntax/strategy/PurchaseOperationTest.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class PurchaseOperationTest {
+    private FruitOperationHandler purchaseOperation;
+    private Map<String, Integer> inventory;
+
+    @BeforeEach
+    void setUp() {
+        inventory = new HashMap<>();
+        purchaseOperation = new PurchaseOperation();
+    }
+
+    @Test
+    void addQuantityToFruit_executeOperation_Ok() {
```

Comment:
The test method name should reflect what it's testing. Consider renaming it to something like 'testSuccessfulPurchaseOperationUpdatesInventoryCorrectly'. This will make it clear what functionality the test covers.